content/altinn-studio/v10/develop-a-service/options/functionality/data-binding/_index.nb.md

```
@@ -1,18 +1,18 @@
 ---
 draft: true
 title: Dataknytning
-description: Hva kan lagres i datamodellen
-
+description: Hva du kan lagre i datamodellen
+weight: 50
 tags: [needsReview, translate]
 ---
 
-### Lagring av valgt alternativ
+### Slik lagrer du valgte alternativer
```

markdownlint-cli2 (0.20.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)